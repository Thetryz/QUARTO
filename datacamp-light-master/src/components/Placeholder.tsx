import * as React from "react";

import * as DatacampLogo from "../images/datacamp-logo.svg";
import * as styles from "./Placeholder.module.scss";

export const Placeholder = () => (
  <div className={styles.placeholder}>
    <DatacampLogo />
  </div>
);

export default Placeholder;
