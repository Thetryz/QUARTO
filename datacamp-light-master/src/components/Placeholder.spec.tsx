import * as React from "react";
import { shallow } from "enzyme";

import { Placeholder } from "./Placeholder";

describe("Placeholder", () => {
  it("should exist", () => {
    expect(Placeholder).toBeDefined();
  });

  it("should match snapshot", () => {
    const tree = shallow(<Placeholder />);
    expect(tree).toMatchSnapshot();
  });
});
