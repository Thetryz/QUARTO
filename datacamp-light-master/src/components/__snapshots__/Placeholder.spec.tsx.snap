// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder should match snapshot 1`] = `
<div
  className="placeholder"
>
  <test-file-stub />
</div>
`;
